pkg/controllers/status/controller.go

```diff
@@ -571,6 +572,7 @@ func (s *StatusController) clusterStatuses(
 			resourceClusterStatus.Error = errMsg
 			clusterStatus = append(clusterStatus, resourceClusterStatus)
 			errList = append(errList, fmt.Sprintf("cluster-name: %s, error-info: %s", clusterName, errMsg))
+			s.recordStatusCollectionError(fedObject.GetName(), fedObject.GetNamespace(), targetGVK.String(), clusterName, errMsg)
```

Comment:
maybe err.Error() is more suitable here？ /cc @limhawjia

Reply:
would both `err.Error()` and `errMsg` be too verbose for a metric tag?

Reply:
let's remove this tag, the error is returned from the underlying informer store and would not be too meaningful

Reply:
done